doc/source/howtos/garbd_howto.rst

```diff
@@ -1,60 +1,66 @@
 .. _garbd_howto:
 
-============================
+=========================================
 Setting up Galera Arbitrator
-============================
+=========================================
 
-`Galera Arbitrator
-<http://galeracluster.com/documentation-webpages/arbitrator.html>`
-is a member of |Percona XtraDB Cluster| that is used for voting
-in case you have a small number of servers (usually two)
-and don't want to add any more resources.
-Galera Arbitrator does not need a dedicated server.
-It can be installed on a machine running some other application.
-Just make sure it has good network connectivity.
+A `Galera Arbitrator
+<https://galeracluster.com/library/documentation/arbitrator.html>`__ 
+is a lightweight member of a **Percona XtraDB Cluster**. This member can vote but does not do any replication and is not included in flow control calculations. The Galera Arbitrator is a separate daemon called ``garbd``. You can start this node separately from the cluster. You cannot configure this node using the ``my.cnf`` file. You can  run this deamon either as a service or from the shell.
 
-Galera Arbitrator is a member of the cluster that participates in the voting,
-but not in actual replication
-(although it receives the same data as other nodes).
-Also, it is not included in flow control calculations.
-
-This document will show how to add Galera Arbitrator node
-to an existing cluster.
+This document will show how to add Galera Arbitrator node to an existing cluster.
```

Comment:
This document SHOWS how to add a Galera Arbitrator node to an existing cluster.

Reply:
Rewritten